docs/dev/table/connect.md

```diff
@@ -1094,6 +1094,23 @@ For append-only queries, the connector can also operate in [append mode](#update
 The connector can be defined as follows:
 
 <div class="codetabs" markdown="1">
+<div data-lang="Java/Scala" markdown="1">
```

Comment:
Java and Scala should be in separate tabs.

Reply:
The scala code is almost the same as Java, so no need to create a separate tab for scala , we have also the case that Java/Scala share the same code demo, such as  Filesystem connectors..

Reply:
Sorry. My mistake.